applications/DEMApplication/python_scripts/DEM_benchmarks_class.py

```diff
@@ -4203,11 +4213,828 @@ def compute_errors(self, velocity_list_outfile_name):  #FINALIZATION STEP
 
         return error1, error2
 
+
+
+class Benchmark40: # multiple benchmarks for general code verification.
+
+    def __init__(self):
+        self.generated_data = None
+        self.balls_graph_counter = 1
+        self.rigid_graph_counter = 1
+
+        self.number_of_DEM_benchmarks = 15
+        self.number_of_FEM_benchmarks = 8
+
+    def ApplyNodalRotation(self, time, dt, modelpart):
+        pass
+
+    def set_initial_data(self, modelpart, rigid_face_model_part, iteration, number_of_points_in_the_graphic, coeff_of_restitution_iteration):
+        pass
+
+
+    def get_final_data(self, modelpart, rigid_face_model_part, cluster_model_part):
+        pass
+
+
+    def generate_graph_points(self, modelpart, rigid_face_model_part, cluster_model_part, time, graph_print_interval, dt):
+
+        #self.graph_frequency = int(5e-7/dt)   #graph_print_interval/dt
+        self.graph_frequency = int(graph_print_interval/dt)   #1 veces mas grf que bin
+        if self.graph_frequency < 1:
+           self.graph_frequency = 1
+
+        if (self.balls_graph_counter == self.graph_frequency):
+            self.balls_graph_counter = 0
+
+            for node in modelpart.Nodes:
+                if node.Id == 10:           ### stage 0 - simple dem
+
+                    force_node_x = node.GetSolutionStepValue(TOTAL_FORCES)[0]
+                    force_node_y = node.GetSolutionStepValue(TOTAL_FORCES)[1]
+                    force_node_z = node.GetSolutionStepValue(TOTAL_FORCES)[2]
+
+                    angular_node_x = node.GetSolutionStepValue(ANGULAR_VELOCITY)[0]
+                    angular_node_y = node.GetSolutionStepValue(ANGULAR_VELOCITY)[1]
+                    angular_node_z = node.GetSolutionStepValue(ANGULAR_VELOCITY)[2]
+
+                    displacement = [0]*3
```

Comment:
you could use 
`def GetDisplacement(node):
     return node.X-node.X0, node.Y-node.Y0, node.Z-node.Z0`
here